static/js/home.js

```diff
@@ -2,11 +2,12 @@
  * Created by sumeetsingharora on 4/16/16.
  */
 
-onscreenMarker = [];
-selectedMarker = [];
+var onscreenMarker = [];
+var selectedMarker = [];
+var roadsegments;
 var map = undefined;
-var heatmap;
-var shift_draw, lastShape;
+var visual_mode;
+var lastShape;
 var facetObj;
 var searchBox;
 var maptToPlot = "heatmap";
```

Comment:
How about move all the map related initialization sub-functions into another javascript file?